cluster/gce/gci/configure-helper.sh

```diff
@@ -1596,17 +1596,17 @@ function start-kube-apiserver {
   if [[ -n "${ENABLE_GARBAGE_COLLECTOR:-}" ]]; then
     params+=" --enable-garbage-collector=${ENABLE_GARBAGE_COLLECTOR}"
   fi
-  if [[ -n "${NUM_NODES:-}" ]]; then
+  if [[ "$(get-num-nodes)" -gt 0 ]]; then
```

Comment:
These scripts are run on the nodes, so I don't think you should change them. It's relying on kube-env to get the right number of nodes. I'm not sure this would actually work.

Given that, to make the env variables consistent, maybe we should still keep `NUM_NODES` as the total number of nodes.

Reply:
Ugh, you're right. I'll update this.

Reply:
Fixed.